dist/husky.js

```diff
@@ -33403,6 +33463,7 @@ define('husky_components/datagrid/decorators/infinite-scroll-pagination',[],func
                 data: null,
                 instanceName: '',
                 searchInstanceName: null,
+                storageName: null,
```

Comment:
the instanceName is not really unique and it should be possible "enabled" independend from `instanceName`. for example if you use it as a contact list in the organization the `instanceName` could be `contacts` but the `storageName` has to be `contacts-of-account-15` to be unique.